### DISCUSSION TITLE
Is Bard's speed the same as in vanilla?

### DISCUSSION THREAD

**Discussion by GrayFace** [diasurgical/DevilutionX#1723] [Q&A] [answered]
https://diablo.fandom.com/wiki/Bard claims that Bard has slow melee attack of the Rogue, but in my game Bard's speed with swords is just as fast as Warrior/Barbarian. Is the Wiki page wrong or did devilutionX change sword speed for Bard?

**Comment by julealgon** [comment, marked as answer]
Table from Jarulf's guide:
![image](https://user-images.githubusercontent.com/461579/116330829-b8b0c600-a7a4-11eb-9beb-5ea8b2cada14.png)

I just tested on vanilla hellfire and in DevilutionX and you are correct: in DevilutionX, the bard seems to be attacking faster with a sword, than the rogue, even though they should have the same speed.

Can you open an issue for this please?

**Reply by GrayFace** [reply]
Here you go https://github.com/diasurgical/devilutionX/issues/1755
